contracts/LostAndFound.cdc

```diff
@@ -503,45 +513,45 @@ pub contract LostAndFound {
             return false
         }
 
-        pub fun setLowBalanceThreshold(threshold: UFix64?) {
+        access(Mutate) fun setLowBalanceThreshold(threshold: UFix64?) {
```

Comment:
Where does the access(Mutate)entitlement get defined? It's not in L&F contract.

Edit: Never mind this is a built-in entitlement (and combines Insert/Remove entitlements)